### DISCUSSION TITLE
v0.17.5

### DISCUSSION THREAD

**Discussion by StevenACoffman** [99designs/gqlgen#2136] [Announcements]
## What's Changed
* Docs: Update federation.md by @lleadbet in https://github.com/99designs/gqlgen/pull/2129
* Ignore protobuf files in coverage by @StevenACoffman in https://github.com/99designs/gqlgen/pull/2133
* update instructions to specify package of Role by @prankymat in https://github.com/99designs/gqlgen/pull/2130
* feat: added graphql.UnmarshalInputFromContext by @giautm in https://github.com/99designs/gqlgen/pull/2131
* Feature: Add FTV1 Support via Handler by @lleadbet in https://github.com/99designs/gqlgen/pull/2132

## New Contributors
* @prankymat made their first contribution in https://github.com/99designs/gqlgen/pull/2130
* @giautm made their first contribution in https://github.com/99designs/gqlgen/pull/2131

**Full Changelog**: https://github.com/99designs/gqlgen/compare/v0.17.4...v0.17.5

<hr /><em>This discussion was created from the release <a href='https://github.com/99designs/gqlgen/releases/tag/v0.17.5'>v0.17.5</a>.</em>

**Comment by ItalyPaleAle** [comment]
Apologies for [repeating myself](https://github.com/99designs/gqlgen/discussions/1769#discussioncomment-2012123), but it would really, really be helpful to have a "human readable" changelog, that explains what's new and how users are impacted.

I have updated from 0.16.0 and noticed that there were some breaking changes (the removal of the `/cmd` package), and it was really hard to figure what I had to update and how. At least for breaking changes, it's very important for us users to have visibility around them and instructions to update our code.

**Reply by StevenACoffman** [reply]
@ItalyPaleAle Yes, that would be great! We would really appreciate you contributing that. Each release has https://github.com/99designs/gqlgen/releases/tag/v0.17.5 release notes attached to it and https://github.com/99designs/gqlgen/blob/master/CHANGELOG.md is generated, but as you pointed out, this is not as gentle of an upgrade path as it could be.

Would you like to start by simply adding your summary to these discussions or make a PR for a new "UPGRADE.md" or something? What did you have in mind?

**Reply by StevenACoffman** [reply]
The transition from v0.14.0 to v0.16.0 to v0.17.0 were particularly big changes, partly because the project had been unmaintained for a long period and there was some fairly major surgery required to stabilize it. So it is worth calling those out in particular.

The removal of `cmd` happened in https://github.com/99designs/gqlgen/pull/1958 

Thanks again for this contribution! It will be much appreciated by all!
